website/content/api-docs/secret/gcp.mdx

```diff
@@ -301,13 +301,190 @@ $ curl \
     https://127.0.0.1:8200/v1/gcp/roleset/my-token-roleset
 ```
 
+## Create/Update Static Account
+
+| Method | Path                        |
+| :----- | :-------------------------- |
+| `POST` | `/gcp/static-account/:name` |
+
+This method allows you to create a static account or update an existing static account. See [docs](/docs/secrets/gcp#bindings) for the GCP secrets backend
+to learn more about what happens when you create or update a static account.
+
+**If you update a static account's bindings, this will effectively revoke any secrets
+generated under this static account.**
+
+### Parameters
+
+- `name` (`string: <required>`): Name of the static account. Cannot be updated.
+- `secret_type` (`string: "access_token"`): Type of secret generated for this static account. Accepted values: `access_token`, `service_account_key`. Cannot be updated.
+- `service_account_email` (`string: <required>`): Email of the GCP service account to manage. Cannot be updated.
+- `bindings` (`string`): Bindings configuration string (expects HCL or JSON format in raw or base64-encoded string). Optional.
+- `token_scopes` (`array: []`): List of OAuth scopes to assign to `access_token` secrets generated under this static account (`access_token` static accounts only)
+
+### Sample Payload
+
+```json
+{
+  "secret_type": "access_token",
+  "service_account_email": "example@mygcpproject.iam.gserviceaccount.com",
+  "bindings": "...",
+  "token_scopes": [
+    "https://www.googleapis.com/auth/cloud-platform",
+    "https://www.googleapis.com/auth/bigquery"
+  ]
+}
+```
+
+#### Sample Bindings:
+
+See [bindings format docs](/docs/secrets/gcp#bindings) for more information.
+
+```hcl
+resource "//cloudresourcemanager.googleapis.com/projects/mygcpproject" {
+  roles = [
+    "roles/viewer"
+  ],
+}
+
+resource "//bigquery.googleapis.com/projects/my-project/datasets/mydataset" {
+  roles = [
+    "roles/bigquery.dataViewer"
+  ],
+}
+
+resource "https://selflink/to/my/resource" {
+  roles = [
+    "project/mygcpproject/roles/projcustomrole",
+    "organizations/myorg/roles/orgcustomrole"
+  ],
+}
+```
+
+### Sample Request
+
+```shell-session
+$ curl \
+    --header "X-Vault-Token: ..." \
+    --request POST \
+    --data @payload.json \
+    https://127.0.0.1:8200/v1/gcp/static-account/my-token-account
+```
+
+## Rotate Static Account Key (`access_token` Static Account Only)
+
+| Method | Path                            |
+| :----- | :------------------------------ | ------------------- |
+| `POST` | `/gcp/static-account/:name/rotate-key` | `204 (empty body)`` |
```

Comment:
```suggestion
| `POST` | `/gcp/static-account/:name/rotate-key` | `204 (empty body)` |
```

Reply:
There are a couple of other places with an extra backtick similar to here.

Reply:
I've fixed those that I have found.